pitch.md

```diff
@@ -14,31 +23,170 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+From time to time, teachers and students of geometry and associated fields are
+interested in creating constructions. While sometimes they understand what they
+are creating, often times they are creating the idea as they make the drawing.
+
+It is also worth noting that while frequently users want these constructions
+rendered into drawings, the objects they are describing are geometric in
+nature - not inherently graphical. This means that sometimes geometrically
+distinct ideas may be spatially close. If input is spatial then this can result
+in geometric imprecision. For example, consider drawing a line from a triangle
+vertex to the midpoint of the opposing side vs bisecting that angle. Often
+these two actions are spatially similar (the lines end up close to each other).
```

Comment:
This is a good point. You mentioned in class that this is where you feel that spacial "click and drag" applications fail to be sufficiently accurate - do you think that a "click and drag" application can be enhanced to be able to provide the precision you want (you could imagine a system that asks which geometrically significant idea you meant when you make an ambiguous line), or do you think addressing these subtleties requires the language to be text-based (as yours is)?